keyboards/yoichiro/lunakey_mini/info.json

```diff
@@ -0,0 +1,56 @@
+{
+    "keyboard_name": "Lunakey Mini",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 14.75,
+    "height": 5,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"label":"L00", "x":0, "y":1.5},
+                {"label":"L01", "x":1, "y":1.25},
+                {"label":"L02", "x":2, "y":0.5},
+                {"label":"L03", "x":3, "y":0},
+                {"label":"L04", "x":4, "y":0.25},
+                {"label":"L05", "x":5, "y":0.5},
+                {"label":"L10", "x":0, "y":2.5},
+                {"label":"L11", "x":1, "y":2.25},
+                {"label":"L12", "x":2, "y":1.5},
+                {"label":"L13", "x":3, "y":1},
+                {"label":"L14", "x":4, "y":1.25},
+                {"label":"L15", "x":5, "y":1.5},
+                {"label":"L20", "x":0, "y":3.5},
+                {"label":"L21", "x":1, "y":3.25},
+                {"label":"L22", "x":2, "y":2.5},
+                {"label":"L23", "x":3, "y":2},
+                {"label":"L24", "x":4, "y":2.25},
+                {"label":"L25", "x":5, "y":2.5},
+                {"label":"L30", "x":3, "y":4},
+                {"label":"L31", "x":4, "y":3.75},
+                {"label":"L32", "x":5, "y":3.75},
+                {"label":"L33", "x":6, "y":4},
+                {"label":"R00", "x":8.75, "y":0.5},
+                {"label":"R01", "x":9.75, "y":0.25},
+                {"label":"R02", "x":10.75, "y":0},
+                {"label":"R03", "x":11.75, "y":0.5},
+                {"label":"R04", "x":12.75, "y":1.25},
+                {"label":"R05", "x":13.75, "y":1.5},
+                {"label":"R10", "x":8.75, "y":1.5},
+                {"label":"R11", "x":9.75, "y":1.25},
+                {"label":"R12", "x":10.75, "y":1},
+                {"label":"R13", "x":11.75, "y":1.5},
+                {"label":"R14", "x":12.75, "y":2.25},
+                {"label":"R15", "x":13.75, "y":2.5},
+                {"label":"R20", "x":8.75, "y":2.5},
+                {"label":"R21", "x":9.75, "y":2.25},
+                {"label":"R22", "x":10.75, "y":2},
+                {"label":"R23", "x":11.75, "y":2.5},
+                {"label":"R24", "x":12.75, "y":3.25},
+                {"label":"R25", "x":13.75, "y":3.5},
+                {"label":"R30", "x":7.75, "y":4},
+                {"label":"R31", "x":8.75, "y":3.75},
+                {"label":"R32", "x":9.75, "y":3.75},
+                {"label":"R33", "x":10.75, "y":4}]
```

Comment:
This block needs to follow the order given by `lunakey_mini.h` – the same order that your `keymap.c` files are in. So `L00 ... L05`, then `R00 ... R05`, then `L10 ... L15`, and so on.

```suggestion
                {"label":"L00", "x":0, "y":1.5},
                {"label":"L01", "x":1, "y":1.25},
                {"label":"L02", "x":2, "y":0.5},
                {"label":"L03", "x":3, "y":0},
                {"label":"L04", "x":4, "y":0.25},
                {"label":"L05", "x":5, "y":0.5},
                {"label":"R00", "x":8.75, "y":0.5},
                {"label":"R01", "x":9.75, "y":0.25},
                {"label":"R02", "x":10.75, "y":0},
                {"label":"R03", "x":11.75, "y":0.5},
                {"label":"R04", "x":12.75, "y":1.25},
                {"label":"R05", "x":13.75, "y":1.5},
                {"label":"L10", "x":0, "y":2.5},
                {"label":"L11", "x":1, "y":2.25},
                {"label":"L12", "x":2, "y":1.5},
                {"label":"L13", "x":3, "y":1},
                {"label":"L14", "x":4, "y":1.25},
                {"label":"L15", "x":5, "y":1.5},
                {"label":"R10", "x":8.75, "y":1.5},
                {"label":"R11", "x":9.75, "y":1.25},
                {"label":"R12", "x":10.75, "y":1},
                {"label":"R13", "x":11.75, "y":1.5},
                {"label":"R14", "x":12.75, "y":2.25},
                {"label":"R15", "x":13.75, "y":2.5},
                {"label":"L20", "x":0, "y":3.5},
                {"label":"L21", "x":1, "y":3.25},
                {"label":"L22", "x":2, "y":2.5},
                {"label":"L23", "x":3, "y":2},
                {"label":"L24", "x":4, "y":2.25},
                {"label":"L25", "x":5, "y":2.5},
                {"label":"R20", "x":8.75, "y":2.5},
                {"label":"R21", "x":9.75, "y":2.25},
                {"label":"R22", "x":10.75, "y":2},
                {"label":"R23", "x":11.75, "y":2.5},
                {"label":"R24", "x":12.75, "y":3.25},
                {"label":"R25", "x":13.75, "y":3.5},
                {"label":"L30", "x":3, "y":4},
                {"label":"L31", "x":4, "y":3.75},
                {"label":"L32", "x":5, "y":3.75},
                {"label":"L33", "x":6, "y":4},
                {"label":"R30", "x":7.75, "y":4},
                {"label":"R31", "x":8.75, "y":3.75},
                {"label":"R32", "x":9.75, "y":3.75},
                {"label":"R33", "x":10.75, "y":4}
            ]
```

Reply:
@noroadsleft Oh, Ok, I didn't know the policy. Thank you, I fix this.